csrc/executor.cpp

```diff
@@ -918,14 +918,20 @@ at::Tensor allocateOutput(
     const FusionExecutor::GlobalBufferInfo& out_info,
     Val* aliased_in,
     const AliasInfo* alias_info,
-    const at::Tensor& aliased_in_tensor,
     const c10::Device& device,
     ExpressionEvaluator& ee) {
   TensorView* out_tv = out_info.tv;
 
   // Note: aliased output is not returned as output. But we still need it
   // for kernel execution, so would need to push them to args
   if (aliased_in != nullptr) {
+    const PolymorphicValue& aliased_in_val = ee.evaluate(aliased_in);
```

Comment:
nitpick, the comment above is out-dated now. i.e. we do want to return aliased output, unless it's just for in-place update.

Reply:
Even for in-place updates, FusionExecutor does return the aliased outputs. It's FusionExecutorCache that hides it (per the `hide_output` flag). So I'll just remove this comment.